client/src/app/containers/Home/ActiveGamesTable.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+import { selectCurrentUser } from '../../../auth/selectors';
+import { authSaga } from '../../../auth/saga';
+import {
+  sliceKey as authSliceKey,
+  reducer as authReducer,
+} from '../../../auth/slice';
+import { selectActiveGames } from '../../../home/selectors';
+import { homeSaga } from '../../../home/saga';
+import {
+  sliceKey as homeSliceKey,
+  reducer as homeReducer,
+  actions as homeActions,
+} from '../../../home/slice';
+
+const ActiveGamesTable = () => {
+  useInjectReducer({ key: authSliceKey, reducer: authReducer });
+  useInjectSaga({ key: authSliceKey, saga: authSaga });
+  useInjectReducer({ key: homeSliceKey, reducer: homeReducer });
+  useInjectSaga({ key: homeSliceKey, saga: homeSaga });
+  const dispatch = useDispatch();
+  const currentUser = useSelector(selectCurrentUser);
+  const activeGames = useSelector(selectActiveGames(currentUser.id));
+
+  // event handlers
+  const onRefreshActiveGames = () => {
+    dispatch(homeActions.refreshActiveGames({ id: currentUser.id }));
+  };
+
+  let gameButtons = [];
+  if (activeGames) {
```

Comment:
Is the server giving us a map here for `activeGames`? I don't think we're actually accessing the games by their keys - would it make more sense to just have the server return a list of games? A list would be easier to consume here on the front end and might make more sense semantically.